zebra-chain/src/parameters/network/testnet.rs

```diff
@@ -669,8 +669,12 @@ impl Parameters {
     pub fn new_regtest(
         ConfiguredActivationHeights { nu5, nu6, nu7, .. }: ConfiguredActivationHeights,
     ) -> Self {
-        #[cfg(any(test, feature = "proptest-impl"))]
-        let nu5 = nu5.or(Some(100));
+        // TODO: Activate Canopy at height 1 at NU5 at height 100 once #9605 is done.
+        let (canopy, nu5) = if cfg!(test) || cfg!(feature = "proptest-impl") {
+            (None, nu5.or(Some(1)))
+        } else {
+            (Some(1), nu5.or(Some(100)))
```

Comment:
We should leave these values as what was passed in except when compiling for tests, notably so that the regular testnet chain up to NU5 activation will continue working.

Reply:
Ah, no, but my comment was also wrong/confusing, it's not possible use the regular Testnet chain with Regtest because it has a different genesis block hash.

I think we still don't want to change the Canopy activation height here except when compiling for tests, so we should only update `let nu5 = nu5.or(Some(100));` to `let nu5 = nu5.or(Some(1));` and leave the rest of this function unchanged.